src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -102,12 +103,34 @@ const MediaContainer = styled.div`
 interface LandingPageProps {
   asset: GenieAsset
   collection: CollectionInfoForAsset
+  setShowDataHeader: (showDataHeader: boolean) => void
 }
 
-export const LandingPage = ({ asset, collection }: LandingPageProps) => {
+export const LandingPage = ({ asset, collection, setShowDataHeader }: LandingPageProps) => {
+  const intersectionRef = useRef<HTMLDivElement>(null)
+
+  // Checks if the intersectionRef is in the viewport
+  // If it is not in the viewport, the data page header becomes visible
+  useEffect(() => {
+    const intersectionHandler = (entries: IntersectionObserverEntry[]) => {
```

Comment:
a potentially nicer way to do this instead of creating a new function and IntersectionHandler on every scroll event is that we can throw it into a ref to be initialized once like this: 

```
  const intersectionRef = useRef<HTMLDivElement>(null)
  const observableRef = useRef(new IntersectionObserver((entries) => {
      if (!entries[0].isIntersecting) {
        setShowDataHeader(true)
      } else {
        setShowDataHeader(false)
      }
  }))

  // Checks if the intersectionRef is in the viewport
  // If it is not in the viewport, the data page header becomes visible
  useEffect(() => {
    const cachedRef = intersectionRef.current
    const observer = observableRef.current
    if (cachedRef && observer) {
      observer.observe(cachedRef)
      return () => observer.unobserve(cachedRef)
    }
    return
  }, [intersectionRef, observableRef, setShowDataHeader])
```